backend/app.ts

```diff
@@ -145,24 +150,21 @@ passport.use(
 
     },
         async function (req, email, password, done) {
-
             const user = await db.user.findUnique({
                 where: {
-                    email: email
-                },
+                    email
+                }
             })
 
-            if (!user || !user.salt) {
-                return done(null, false);
+            if (!user?.salt) {
+                done(null, false); return;
```

Comment:
this looks like a typo?

Reply:
When I leave it like this:
if (!user.salt) {
      done(null, false);

I get this error for the user word -> 'user' is possibly 'null'.

Reply:
Would including a check if the user exists at all be a good fix such as:

if (!user) {
      done(null, false);
      return;
}

Reply:
Update: This was fixed after I pulled hyuns latest push to main.